kevinhartmann23/fitlit
# Initial commit

#### What does  this PR do?

This PR begins to implement TDD by creating a sample test data file, as well as test files.

#### Where should the reviewer start?

Familiarize yourself with the test data file, as well as the test files and how this data is being implemented.

#### Is this a bug fix or a feature?

Feature.

#### How should this be manually tested?

In terminal, run first test files to ensure that mocha/chai are functioning correctly.
